app/jobs/email_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class EmailJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    unseen_comments = Comment.unseen_comments
+    return if unseen_comments.empty?
+
+    unique_users = unseen_comments.map(&:user).uniq
```

Comment:
bad code, because
1. you load all unseen comments
2. then you go through it and get user, it makes new request to database every time, it's n+1
You should load users by joins
read https://guides.rubyonrails.org/active_record_querying.html#joining-tables